pyxem/generators/vdf_generator.py

```diff
@@ -71,11 +73,64 @@ def get_vdf_images(self,
         vdfs : Signal2D
             Signal containing virtual dark field images for all unique vectors.
         """
+        if self.vectors:
+            vdfs = []
+            for v in self.vectors.data:
+                disk = roi.CircleROI(cx=v[0], cy=v[1], r=radius, r_inner=0)
+                vdf = disk(self.signal,
+                           axes=self.signal.axes_manager.signal_axes)
+                vdfs.append(vdf.sum((2,3)).as_signal2D((0,1)).data)
+
+            vdfim = VDFImage(np.asarray(vdfs))
+
+            if normalize==True:
+                vdfim.map(normalize_vdf)
+
+        else:
+            raise ValueError("DiffractionVectors non-specified by user. Please "
+                             "initialize VDFGenerator with some vectors. ")
+
+        return vdfim
+
+    def get_concentric_vdf_images(self,
+                                  k_min,
+                                  k_max,
+                                  k_steps,
+                                  normalize=False):
+        """Obtain the intensity scattered at each navigation position in an
+        ElectronDiffraction Signal by summation in an
+        annlus of specified inner and outer radius.
```

Comment:
This should reflect the fact that multiple images are formed. E.g.

"....by summation over a series of concentric annuli between a specified inner and outer radius in a number of steps."